zyte_common_items/base.py

```diff
@@ -16,7 +16,13 @@
 
 from typing import Dict, List, Optional, Union
 
-from zyte_common_items.util import export, split_in_unknown_and_known_fields
+import attrs
+from itemadapter import ItemAdapter
+
+from zyte_common_items.util import (
+    _remove_empty_values,
+    split_in_unknown_and_known_fields,
+)
 
 
 def is_data_container(cls_or_obj):
```

Comment:
Minor and optional: we could probably move this to `util.py`.

Reply:
It depends on the `Item` class defined below, so it would not be trivial.

Reply:
We could redefine it as a class method of `Item`, e.g. `is_instance_or_subclass`.

Reply:
Let's leave it as-is for now.